src/api/java/com/minecolonies/api/colony/permissions/Action.java

```diff
@@ -0,0 +1,57 @@
+package com.minecolonies.api.colony.permissions;
```

Comment:
I feel like we shouldn't add api to the package, that way everything can stay in the same packages. The code will just exist in different modules